app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -120,6 +120,11 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         default = 0L
     )
 
+    var lastBrowserActivity by longPreference(
```

Comment:
Can you add some kDoc for this? I think in general we want to add kDocs for any new Settings since there are so many of them and we want to preserve some knowledge.

Reply:
Sure 👍🏽